packages/transport/README.md

```diff
@@ -27,8 +27,8 @@ firmware repo are semi-automatically translated to Flow and TypeScript definitio
 In order to be able to use new features of trezor-firmware you need to update protobuf definitions.
 
 1. `git submodule update --init --recursive` to initialize trezor-common submodule
-1. `yarn workspace @trezor/transport update:submodules` to update trezor-common submodule
-1. `yarn workspace @trezor/transport update:protobuf` to generate new `messages.json` and `protobuf.d.ts`
+2. `yarn workspace @trezor/transport update:submodules` to update trezor-common submodule
+3. `yarn workspace @trezor/transport update:protobuf` to generate new `messages.json` and `protobuf.d.ts`
```

Comment:
not sure about `protobuf.d.ts`. i think that the output file got renamed recently

Reply:
Fixed in 890d28905940c10e8c72c9f5f9d9ecb48b6d8007.